pt_PT.ini

```diff
@@ -10,52 +10,52 @@ Sound speed hack (DOA etc.) = Sound speed hack (Dead Or Alive, etc.)
 WASAPI (fast) = WASAPI (fast)
 
 [Controls]
-Analog Limiter = Analog limiter
-Analog Mapper High End = Analog mapper high-end (axis sensitivity)
-Analog Mapper Low End = Analog mapper low-end (inverse deadzone)
-Analog Mapper Mode = Analog mapper mode
-Analog Stick = Analog stick
+Analog Limiter = Limitador do analógico
+Analog Mapper High End = Mapeamento do analógico de alta qualidade (sensibilidade dos eixos)
+Analog Mapper Low End = Mapeamento do analógico de baixa qualidade (zona não coberta inversa)
+Analog Mapper Mode = Modo de mapamento do analógico
+Analog Stick = Analógico
 Auto = Auto
-Auto-centering analog stick = Auto-centering analog stick
-Button Opacity = Buttons opacity
-Button style = Button style
-Calibrate D-Pad = Calibrate
-Calibration = Calibration
-Classic = Classic
-Control Mapping = Control mapping
-Custom layout... = Edit touch control layout...
-Customize tilt = Customize tilt...
+Auto-centering analog stick = Auto-centralização do analógico
+Button Opacity = Opacidade dos botões
+Button style = Estilo dos botões
+Calibrate D-Pad = Calibrar D-Pad
+Calibration = Calibração
+Classic = Clássico
+Control Mapping = Mapeamento de Controlos
+Custom layout... = Editar layout dos controlos de toque...
+Customize tilt = Personalizar inclinação...
 D-PAD = D-Pad
-Deadzone Radius = Deadzone radius
-DInput Analog Settings = DInput analog settings
-Disable D-Pad diagonals (4-way touch) = Disable D-Pad diagonals (4-way touch)
-HapticFeedback = Haptic feedback (vibration)
-Ignore gamepads when not focused = Ignore gamepads when not focused
-Ignore Windows Key = Disable Windows key
-Invert Axes = Invert axes
-Invert Tilt along X axis = Invert tilt along X axis
-Invert Tilt along Y axis = Invert tilt along Y axis
-Keyboard = Keyboard control settings
-Landscape = Landscape
-Landscape Reversed = Landscape reversed
-None (Disabled) = None (disabled)
+Deadzone Radius = Raio não coberto
+DInput Analog Settings = Definições de analógico DInput
+Disable D-Pad diagonals (4-way touch) = Desactivar diagonais do D-Pad (4 direcções)
```

Comment:
`touch` is not being taken into account on the translation but, honestly, I think it is good as it is :+1: